docs/setup/install-with-binary.md

```diff
@@ -2,16 +2,19 @@
 title: Installing KubeEdge with Binary
 sidebar_position: 4
 ---
-Deploying KubeEdge with binary is used to test, never use this way in production environment.
+
+Deploying KubeEdge with binary is used for testing purposes and should not be used in a production environment.
 
 ## Limitation
 
-- Need super user rights (or root rights) to run.
+- It Requires super user rights (or root rights).
 
 ## Setup Cloud Side (KubeEdge Master Node)
 
 ### Create CRDs
 
+Before setting up the cloud side, you need to create the necessary Custom Resource Definitions (CRDs) for KubeEdge. These CRDs define the custom resources that KubeEdge uses for managing devices, applications, and other components. To create the CRDs, run the following commands:
+
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha2_device.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha2_devicemodel.yaml
```

Comment:
```suggestion
Before setting up the cloud side, you need to create the necessary Custom Resource Definitions (CRDs) for KubeEdge. These CRDs define the custom resources that KubeEdge uses for managing devices, applications, and other components. To create the CRDs, run the following commands:

```shell
kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1beta1_device.yaml
kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1beta1_devicemodel.yaml
```

Reply:
After release 1.15, we use the beta version of device crd, which can be updated here.